src/scripts/components/VideoListItem.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import type { UploadRecord } from 'records/UploadRecords'
+
+type Props = {
+  id: string,
+  item: UploadRecord,
+  onClick: (id: string) => void
+}
+
+const Item = styled.div`
+  background-color: ${props => props.theme.colors.body.background};
+  font-family: ${props =>
+    props.theme.fonts.family ? props.theme.fonts.family : 'Monospace'},
+    sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid grey;
+  padding: 10px;
+`
+
+const Label = styled.p`
+  color: white;
```

Comment:
Please use the colors @jrgarou defined. Example [here](https://github.com/Paratii-Video/paratii-portal/blob/3f1abb724d25836dc49858869f226e2e6cd4c951/src/scripts/components/structures/Main.js#L9), and you may need/want to edit the [themes file](https://github.com/Paratii-Video/paratii-portal/blob/3f1abb724d25836dc49858869f226e2e6cd4c951/src/scripts/components/foundations/base/Themes.js)